src/table/table.scss

```diff
@@ -315,17 +235,12 @@
       }
 
       // Added by Jon, not fully tested
-      + .iui-expanded-content {
+      + .iui-expandable-content {
```

Comment:
Why change this class? Now it has the possibility of conflicting with expandable block

Reply:
lol I changed it to be more consistent with the expandable block. 😅

Reply:
Changed it back